sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -58,6 +64,46 @@
 neighbors.radius_neighbors_graph = ignore_warnings(neighbors.radius_neighbors_graph)
 
 
+def _get_dummy_metric_params_list(metric: str, n_features: int):
```

Comment:
nitpick: `_generate_test_params_for` would be more descriptive. I cannot help to parse the "dummy metric" phrase otherwise.

Reply:
That's right. I must use another phrasing.